crmsh/bootstrap.py

```diff
@@ -99,8 +99,9 @@ def __init__(self):
         self.qdevice_heuristics_mode = None
         self.qdevice_rm_flag = None
         self.qdevice_reload_policy = QdevicePolicy.QDEVICE_RESTART
-        self.shared_device = None
-        self.ocfs2_device = None
+        self.ocfs2_devices = []
+        self.use_clvm2 = None
```

Comment:
s/_clvm2/_cluster_lvm2/g

since clvm is a legacy name used widely in the past

Reply:
I prefer to keep this unchanged, _clvm2 was short for cluster lvm2, and it could keep consistent while backporting to 12sp4+
Anyway, it only used internally, not export to users

Reply:
I have less vision to do backport. 12sp4+ is just too old. It's a "burden" to slow us down.

Reply:
I undestand this is not a end-user related topic. However, I insist to get rid of _clvm2. I know how painful to clarify the terminologies in related to clvm alone. I can imagine the pain if some other developer starts to have the question about it when deals with the code around.

Reply:
OK, changed